{{ cookiecutter.package_name }}/rtd-pip-requirements

```diff
@@ -0,0 +1,5 @@
+Cython
```

Comment:
Why don't we move this file to `docs/requirements.txt` or `docs/rtd-requirements.txt`? I feel like we don't need another file in the root of the repo.

Reply:
I can move it as long as there is a consensus. I feel people have been flip-flopping where that file should go throughout the years and we move it back and forth.

Reply:
I don't have a preference, but picking one and sticking to it in all the repos would be nice. So far astropy and photutils have `docs/rtd_requirements.txt`

Reply:
Okay, I moved and renamed it. Also squashed commits.